azavea/pfb-network-connectivity
# Feature/awf/django overall scores

## Overview

Add overall_scores field to AnalysisJob as jsonb, and push analysis results from CSV back to Django app, so we can easily sort/filter on these scores later.

### Demo

<img width="1253" alt="screen shot 2017-03-24 at 10 50 20" src="https://cloud.githubusercontent.com/assets/1818302/24299772/a2d3cd10-1080-11e7-8f9d-c1be0ae6045a.png">

## Testing Instructions

- Start new AnalysisJob for Germantown, either via API UI or Django shell
- Copy the run-local-analysis command printed by AnalysisJob.run() in the django docker console
- Run job in VM
- When job is finished, overall_scores should have data for the job id you just ran, inspect in Django console
- Overall scores should not print in analysis_jobs list view in API

Closes #185
